### DISCUSSION TITLE
[Question] window-controls-overlay mock compatibility

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#371] [Q&A]
Hey, would it be possible to recreate the window-controls-overlay API in this, or is that out of the realm of possibility?

**Comment by DustinBrett** [comment]
It would be cool and it is for sure a long term plan to be able to do that, but it's only for PWA's so before I can do anything I have a long term plan to move to a PWA. Also it still has quite early days support for Chromium 105+ which came out about a year ago now. https://developer.mozilla.org/en-US/docs/Web/API/Window_Controls_Overlay_API#browser_compatibility

**Reply by CorruptionVR** [reply]
No, what I mean is if you could recreate the functionality in the OS.JS desktop environment. For example, daedalOS has a browser, but it is currently only able to have one page and doesn't have any tabs. Recreating the window-controls-overlay API would allow the ability to put a tab strip in line with the minimize, maximize, and close buttons, just like actual Windows.

**Reply by DustinBrett** [reply]
I do indeed plan to add tabs to the Browser if that is what you are getting at.
